sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/testing/TableContainer.java

```diff
@@ -40,14 +41,14 @@
   long addRow(TableRow row, String id) {
     rows.add(row);
     ids.add(id);
-    long rowSize = row.toString().length();
-    Long tableSize = table.getNumBytes();
-    if (tableSize == null) {
-      table.setNumBytes(rowSize);
```

Comment:
`table.setNumBytes` was called whether tableSize was null or not. So extracted the method call outside if statement.